### DISCUSSION TITLE
useLocation parent base url

### DISCUSSION THREAD

**Discussion by jamalsoueidan** [molefrog/wouter#480] [Q&A] [answered]
Hi

Thanks for wouter, im using it in small app.

But wondering how useLocation works in relation to base url, I have few of the base <Route base="...">

I'm trying to do this setLocation('../'), not working..I need to get out of the closest base, if i use setLocation("/") it goes back to the parent base url, which is great, but how to get out of that to the parent base url?

**I cant use ~ since it will remove all the base urls.**.

Thanks

**Comment by molefrog** [comment, marked as answer]
> I'm trying to do this setLocation('../'), not working..I need to get out of the closest base, if i use setLocation("/") it goes back to the parent base url, which is great, but how to get out of that to the parent base url?

Well, this is currently not supported and it will be quite tricky to implement in current architecture since we don't link current router with its parent.
